controller/namedspaces.go

```diff
@@ -62,14 +62,19 @@ func (c *NamedspacesController) List(ctx *app.ListNamedspacesContext) error {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(fmt.Sprintf("not found, userName=%v", ctx.UserName)))
 	}
 
+	sort, err := space.ParseSortSpaceBy(ctx.Sort)
+	if err != nil {
+		return err
```

Comment:
The error return value from controller functions should wrap the original error object  with `jsonapi.JSONErrorResponse` function call.

Reply:
Thanks, changed!